spec/spec_helper_acceptance.rb

```diff
@@ -31,20 +45,15 @@
     run_puppet_access_login(user: 'admin') if pe_install? && (Gem::Version.new(puppet_version) >= Gem::Version.new('5.0.0'))
     hosts.each do |host|
       # This will be removed, this is temporary to test localisation.
-      if (fact('osfamily') == 'Debian' || fact('osfamily') == 'RedHat') &&
```

Comment:
This localization code may be required, please double check in CI that localization tests run with no issues.